src/game/server/gamemodes/base_pvp/base_pvp.cpp

```diff
@@ -767,6 +767,8 @@ int CGameControllerPvp::OnCharacterDeath(class CCharacter *pVictim, class CPlaye
 {
 	CGameControllerDDRace::OnCharacterDeath(pVictim, pKiller, Weapon);
 
+	const bool SuicideOrWorld = Weapon != WEAPON_SELF && Weapon != WEAPON_WORLD;
```

Comment:
Isn't this the opposite of SuicideOrWorld? Seems to me like the variable name is misleading.

Reply:
It is, my bad, 3 AM coding hits different